tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -2045,6 +2046,26 @@ def test_comparison_alert_below(self):
             incident, [self.action], [(50.0, IncidentStatus.CLOSED, mock.ANY)]
         )
 
+    @with_feature("organizations:metric-alert-ignore-archived")
+    def test_is_unresolved_query(self):
+        rule = self.comparison_rule_above
+        update_alert_rule(rule, query="(event.type:error) AND (is:unresolved)")
```

Comment:
Can we add a note here to call out how this explicit query affects the expected results?

Reply:
eg. utilizes the `ErrorsQueryBuilder` and requires an entity or something